.github/workflows/release-drafter.yaml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     # branches to consider in the event; optional, defaults to all
     branches:
-      - master
+      - main
```

Comment:
this one I fixed manually actually; we should fix this on starbase

Reply:
Done! https://github.com/canonical/starbase/pull/78